source/Services/xdr-agent-crime.html.md.erb

```diff
@@ -0,0 +1,189 @@
+
+To trigger the Ansible, we have created two pipelines in the main Crime Jenkins instance which can be scope limited by Ansible's Limit functionality. This enables us to follow a rollout plan through environments and safely stage our delivery.
+
+When the deploy pipeline is run, this calls an Ansible playbook that includes a Role we have developed which handles the installation.
+
+This document details the procedure to do this.
+
+
+
+## Ansible Inventory/Variables
+
+Before running the pipelines, two variables need declaring in either Ansible group_vars or host_vars. These are:
+
+- cortex_env
+- xdr_tags
+
+cortex_env is a string which needs set to either 'nonprod' or 'prod'. MoJ SoC provided a separate installation package for nonprod & prod, this defines which installation package is used.
+
+xdr_tags is a comma-separated string of tags to provide to the Cortex Agent.
+
+Example:
+
+```bash
+xdr_tags: "hmcts,server,idam"
+cortex_env: "nonprod"
+```
+
+An MS Team channel exists called "HMCTS - Tagging Catch Up" with the MoJ SoC team as members. Please reach out for help if unsure on tags on VMs.
+
+### Where to set these?
+
+These need setting in the automation.ansible repository. The vars live at /sp-ansible/group_vars & /sp-ansible/host_vars. Information on Ansible variables is available in the [official documentation](https://docs.ansible.com/archive/ansible/2.3/intro_inventory.html).
+
+Its possible the level at which you wish to set these variables doesn't have a pre-existing vars file. e.g. there may not be a host_vars/X file for your host or a group_vars/X file for the group level. If you find this, create the necessary vars file.
+
+### Other Role variables
+
+Other variables are defaulted within the Role and do not need setting in automation.ansible repo inventory. The only exception to this is 'sa_key'. This has been set on the 'all' group and is already usable by all hosts.
+
+
+## Deploying the XDR agents
+
+### Pipelines
+
+The pipelines in Jenkins for deployment are:
+
+- xdr-ansible-inventory
+- xdr-ansible-deploy
+
+xdr-ansible-deploy is the pipeline which does the actual installation. xdr-ansible-inventory queries Azure and brings back JSON data used to populate the parameter options for xdr-ansible-deploy.
+
+xdr-ansible-inventory is set on a schedule to run each morning and update its backend JSON data. This schedule should be enough to ensure groups and host parameter options are up-to-date with Azure. 
+
+```
+Caveat: If new VMs are deployed during the day and the rollout plan after this, the new VMs will not be included in the parameter option list. This isn't likely as new VMs are rarely deployed. If they are the pipeline can be re-run ad-hoc without issue to get the latest data. Make sure this is only run when xdr-ansible-deploy is NOT running! 
+```
+
+
+### Using the Pipelines
+
+If you choose to, run the xdr-ansible-inventory pipeline first. There are no build parameters to this, simply click 'Build now'.
```

Comment:
Want to make this a "run this pipeline first" instead of a choice? Sounds like it'll be scheduled or a dependent build in the future, and good practice to keep machine list up to date anyway